doc/Type/independent-routines.pod6

```diff
@@ -1235,17 +1235,17 @@ say shift @a; # ERROR: «Cannot shift from an empty Array[Int]␤»
 
 Defined as:
 
-    multi sub push(\container, **@values is raw)
-    multi sub push(\container, \arg)
+    multi sub push(\collection, **@values is raw)
```

Comment:
Ditto here. It uses \a as an argument.

Reply:
Ditto here. An example of a non-`Positional` collection type implementing `.pop` would be a heap.

Reply:
In this case using `\a` would really make the signature less meaningful.

Reply:
Well, it's what it is. If we want to get really picky about it, it's `\object_can_push`. Which is why in the absence of a consensus, it's really the best to stick to the source. But, again, your call. We at least agree that the original `\container` was not a good idea.